src/psyclone/tests/psyir/transformations/kernel_transformation_test.py

```diff
@@ -460,13 +367,23 @@ def test_kernel_trans_validate(monkeypatch):
     kernels = sched.walk(Kern)
     kernel = kernels[0]
 
-    def dummy_func():
-        '''Simple Dummy function that raises SymbolError.'''
+    def raise_symbol_error():
+        '''Simple function that raises SymbolError.'''
         from psyclone.psyir.symbols import SymbolError
         raise SymbolError("error")
-    monkeypatch.setattr(kernel, "get_kernel_schedule", dummy_func)
+    monkeypatch.setattr(kernel, "get_kernel_schedule", raise_symbol_error)
     with pytest.raises(TransformationError) as err:
         kernel_trans.apply(kernel)
     assert ("'kernel_with_global_code' contains accesses to data that are "
-            "not captured in the PSyIR Symbol Table(s) (error)."
-            "" in str(err.value))
+            "not present in the Symbol Table(s). Cannot transform such a "
+            "kernel." in str(err.value))
+
+    def raise_gen_error():
+        '''Simple function that raises GenerationError.'''
+        from psyclone.errors import GenerationError
```

Comment:
Please could you move this import (and the one earlier for SymbolError) up to the top to improve pylint a bit.

Reply:
Actually, I realise now that pylint says that `GenerationError` has already been imported.

Reply:
Moved all imports to the top and ordered them alphabetically